mmgen/models/architectures/biggan/modules.py

```diff
@@ -364,7 +363,7 @@ def __init__(self,
                 kernel_size=1,
                 stride=1,
                 padding=0,
-                conv_cfg=conv_cfg,
+                conv_cfg=shortcut_cfg,
```

Comment:
Will this shortcut_cfg be changed?

Reply:
In the original implementation, the answer is no. I remain this for users to justify their models.

Reply:
Any update?